src/app/beer_garden/local_plugins/manager.py

```diff
@@ -692,6 +689,16 @@ def _process_args(plugin_config: Dict[str, Any], instance_name: str):
         if plugin_args is not None:
             process_args += plugin_args
 
+        if plugin_config["AUTO_BREW_ARGS"]:
+            plugin_auto_args = plugin_config["AUTO_BREW_ARGS"].get(instance_name)
+            if plugin_auto_args is not None:
+                process_args += ["ARG=" + arg for arg in plugin_auto_args]
+
+        if plugin_config["AUTO_BREW_KWARGS"]:
+            plugin_auto_kwargs = plugin_config["AUTO_BREW_KWARGS"].get(instance_name)
+            if plugin_auto_kwargs is not None:
+                process_args += ["KWARG=" + kwarg for kwarg in plugin_auto_kwargs]
```

Comment:
How is the key being set here?

Reply:
Fixed. I think plugin_auto_kwargs.items() is cleaner but I do like your solution.